cmd/swarmctl/service/inspect.go

```diff
@@ -80,7 +80,7 @@ func printServiceSummary(service *api.Service, running int) {
 		}
 	}
 	if len(service.Spec.Task.Networks) > 0 {
-		fmt.Fprintf(w, "  Networks:\t")
+		fmt.Fprintln(w, "  Networks:\t")
```

Comment:
I notice other similar places, like line 64,74,78,etc, is used "Fprintln".  Need I replace all of them or only replace the pointed out? @aaronlehmann